demos/index.rst

```diff
@@ -16,3 +16,5 @@ Samples and Demos
    ../open-amp/docs/apps/echo_test/README
    ../open-amp/docs/apps/matrix_multiply/README
    ../open-amp/docs/apps/rpc_demo/README
+   ./system_reference-AMD-Xilinx.rst
+   ./system_reference-ST.rst
```

Comment:
Do these instructions already exist in the system reference repo in a readme file?  If so, I think it would be a good practice to link to an existing document so we don't have to maintain two versions.

Reply:
The instructions for AMD-Xilinx are here: https://github.com/OpenAMP/openamp-system-reference/tree/main/examples/linux/rpmsg-echo-test/README.md

each demo has its own README.md

Reply:
Total 3 demo: rpmsg-echo-test, rpmsg-mat-mul, rpmsg-proxy-app

Reply:
@TanmayShah-xilinx  Thank you.  I actually created a ticket to review the examples in the system reference repo and decide whether they should be pulled into the docs instead of or in addition to the content that is currently being pulled from the Wiki.  Please see https://github.com/OpenAMP/openamp-docs/issues/5